AdminWebsite/AdminWebsite/ClientApp/src/app/services/pipe-stringifier.service.ts

```diff
@@ -9,7 +9,7 @@ export class PipeStringifierService {
     encode<T>(input: T): string {
         let output = '';
         for (const property in input) {
-            if (Object.prototype.hasOwnProperty.call(input, property)) {
+            if (Object.prototype.hasOwnProperty.call(input, property) && !!input[property]) {
```

Comment:
Dev Api seems to not like getting the following
|JudgeEmail||JudgePhone||OtherInformation|something

This would mean that would return
|OtherInformation|something